src/pathfinding_service/service.py

```diff
@@ -285,6 +288,19 @@ def handle_channel_closed(self, event: ReceiveChannelClosedEvent) -> None:
             )
             metrics.get_metrics_for_label(metrics.ERRORS_LOGGED, metrics.ErrorCategory.STATE).inc()
 
+    def handle_channel_settled(self, event: ReceiveChannelSettledEvent) -> None:
```

Comment:
`handle_channel_settled` and `handle_channel_closed` could be the same function if we are willing to sacrifice (or special case) the error logging in `handle_channel_closed`.

Reply:
I reworked it a bit, let me know if this is better.

If we want we could also store if the channel was closed in the DB and then act on this. But it feels a bit over-engineered for this case.